### DISCUSSION TITLE
Welcome to xfinity-data-usage Discussions!

### DISCUSSION THREAD

**Discussion by zachowj** [zachowj/xfinity-data-usage#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vmsman** [comment]
I am using the latest for xfinity-data-usage.  As of late,  it appears that a login is not occuring.  Any ideas?
`
xfinity  | 2024-03-30 19:28:13 [info] : Xfinity Update every 120 mins
xfinity  | 2024-03-30 19:28:13 [info] : Http server will be started
xfinity  | 2024-03-30 19:28:13 [info] : Will post to http://localhost:8000 on new data
xfinity  | 2024-03-30 19:28:13 [info] : http server started
xfinity  | 2024-03-30 19:28:13 [verbose] : Fetching Data
xfinity  | 2024-03-30 19:28:14 [debug] : Loading https://customer.xfinity.com/#/devices#usage
xfinity  | 2024-03-30 19:28:20 [debug] : Current URL: https://login.xfinity.com/login?c_ds_na=c_ds_na%2Cc_ds_no%2Cc_ds_ts%2Cclient_id%2Ccontinue%2CforceAuthn%2Cr%2CreqId%2Cs&c_ds_no=F954F083EE477B746C586161E608D565&c_ds_ts=1711844897&c_ds_val=11F4F9FA49654BD67DCE58D0BDAA40FE555B7D06F598996A0BDE1AB0B2EDB1A6&client_id=my-account-web&continue=https%3A%2F%2Foauth.xfinity.com%2F%2Foauth%2Fauthorize%3Fclient_id%3Dmy-account-web%26prompt%3Dlogin%26redirect_uri%3Dhttps%253A%252F%252Fcustomer.xfinity.com%252Foauth%252Fcallback%26response_type%3Dcode%26state%3Dhttps%253A%252F%252Fcustomer.xfinity.com%252F%2523%252Fdevices%2523usage%26response%3D1%26reqId%3Dab5a60a5-64bc-4c36-9eb0-670b9aa778c3&forceAuthn=1&r=comcast.net&reqId=4b233922-817d-4ab3-a804-7988150025fd&s=oauth
xfinity  | 2024-03-30 19:28:20 [debug] : Filling in username
xfinity  | 2024-03-30 19:28:22 [debug] : Current URL: https://login.xfinity.com/login
xfinity  | 2024-03-30 19:28:22 [debug] : Filling in username
xfinity  | 2024-03-30 19:28:24 [debug] : Current URL: https://login.xfinity.com/login
xfinity  | 2024-03-30 19:28:24 [debug] : Filling in username
xfinity  | 2024-03-30 19:28:26 [debug] : Current URL: https://login.xfinity.com/login
xfinity  | 2024-03-30 19:28:26 [error] : Error: State did not change for 3 tries. Last state: NotLoggedIn. Next update at 9:28:13 PM

`
